cypress/support/commands.js

```diff
@@ -25,17 +25,52 @@
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
 
 import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
+import faker from 'faker';
+import { generateNewArticle } from './generateNewArticle';
+import { generateNewUser } from './generateNewUser';
 
 addMatchImageSnapshotCommand();
 
+Cypress.Commands.add(`findByPlaceholder`, (placeholder) => {
+  cy.get(`[placeholder = "${placeholder}"]`);
+});
+
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[data-qa="${selector}"]`);
+});
+
+Cypress.Commands.add(`registerNewUser`, () => {
+  const user = generateNewUser();
+
+  cy.request('POST', 'http://localhost:1667/users', user)
+      .then(response => ({...response.body.user, ...user}));
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
-    password
-  });
+Cypress.Commands.add(`login`, (user) => {
+  cy.request('POST', 'http://localhost:1667/users/login', {
+      user: {
+          email: user.email,
+          password: user.password
+      }
+  })
+      .then(response => {
+          cy.setCookie('drash_sess', response.body.user.token )
```

Comment:
The 'login' command does not return a value. Consider returning the response for further use or checks in tests.